platform/platform-logging/src/main/java/org/codice/ddf/logging/platform/LogEvent.java

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.logging.platform;
+
+import org.apache.commons.lang.builder.EqualsBuilder;
+import org.apache.commons.lang.builder.HashCodeBuilder;
+
+/**
+ * Describes a log event in the system
+ */
+public class LogEvent implements Comparable<LogEvent> {
+
+    private final long timestamp;
+
+    private final String level;
```

Comment:
Shouldn't we use an enum here so we can have a well-defined set of levels? That would also ensure that clients know ahead of time what levels are possible and things are consistent.